services/system/Transact/src/RTransact.cpp

```diff
@@ -1443,12 +1444,23 @@ std::optional<AccountNumber> RTransact::getUser(HttpRequest request)
             return result;
       }
    }
-   if (auto token = request.getCookie("__Host-SESSION"))
+
+   std::string_view cookieName = "__Host-SESSION";
+   if (isDevChain)
+   {
+      cookieName = "SESSION";
+   }
+
+   if (auto token = request.getCookie(cookieName))
```

Comment:
It's possible to have multiple values for a cookie (`__Host` prevents this, so it was fine before).

Reply:
I don't see any evidence of multiple values for cookies (sort of custom encoding of such). RFC 6265 seems clear about that and RFC 6265big doesn't seem to have any reference to the contrary. Could you point me at what you're looking at?

Reply:
Ok, this PR used the pre-existing getCookie(), and given the scope of this PR, I've created an issue to address this issue by upgrading getCookie() to be spec compliant #1337